tests/testdata/default_config.yaml

```diff
@@ -260,6 +260,10 @@ train:
   # Allow CPU offload for deepspeed optimizer.
   # Default: False
   deepspeed_cpu_offload_optimizer: false
+  # Whether or not we should disable the use of flash-attention during training.
```

Comment:
this is good text to also populate into the flag. Or does the clickext class do this?

Reply:
I believe @JamesKunstle mentioned that clickext does this automatically, but I'll double-check.

Reply:
Yeah, it's added automatically here: [github.com/RobotSail/instructlab](https://github.com/RobotSail/instructlab/blob/dedf9b43ca27b06ad5798bab704997f754e8af6a/src/instructlab/configuration.py#L639)